resource/js/docready.js

```diff
@@ -96,6 +96,12 @@ $(function() { // DOCUMENT READY
     });
 
     var active_tab = $('a.active').parent().attr('id');
+
+    active_tab == 'alpha' ? $('#alpha').css('pointer-events','none') : $('#alpha').css('pointer-events','auto');
+    active_tab == 'changes' ? $('#changes').css('pointer-events','none') : $('#changes').css('pointer-events','auto');
+    active_tab == 'groups' ? $('#groups').css('pointer-events','none') : $('#groups').css('pointer-events','auto');
+    active_tab == 'hierarchy' ? $('#hierarchy').css('pointer-events','none') : $('#hierarchy').css('pointer-events','auto');
+
```

Comment:
Would this achieve the same result?

```
    var active_tab = $('a.active').parent().attr('id');
    $('#'+active_tab).css('pointer-events','none') ;
```

Reply:
The code you propose is more elegant and we will go with it